tests/v2/validation/deleting/README.md

```diff
@@ -26,4 +26,7 @@ These tests utilize Go build tags. Due to this, see the below examples on how to
 ### RKE2 | K3S
 `gotestsum --format standard-verbose --packages=github.com/rancher/rancher/tests/v2/validation/deleting --junitfile results.xml -- -timeout=60m -tags=validation -v -run "TestClusterDeleteTestSuite/TestDeletingCluster"`
 
+### Delete Init Machine Suite (rke2/k3s)
```

Comment:
can you add a little here about why we have this test/ more about what it is testing? I don't think we do this enough yet but we're adding to the READMEs a lot more now so this clarity is necessary in my opinion.

Reply:
sure thing  -- great idea 💡